packages/plugins/type-designer/src/components/elements-type-container/element-type-container.svelte

```diff
@@ -1,17 +1,19 @@
 {#if $xmlDocument}
-	<ToolBar />
+	<ContentTabs {service} />
 {:else}
 	<p>No xml document loaded</p>
 {/if}
 
 <script lang="ts">
-// COMPONENTS
-import ToolBar from './tool-bar.svelte'
-// STORES
-import { xmlDocumentStore } from '../../stores'
+	import { DataTypeTemplatesService } from "@oscd-plugins/core";
 
-//stores
-const { xmlDocument } = xmlDocumentStore
-</script>
+	// STORES
+	import { xmlDocumentStore } from "../../stores";
+	import ContentTabs from "./columns.svelte";
+	import { get } from "svelte/store";
 
+	//stores
+	const { xmlDocument } = xmlDocumentStore;
 
+	const service = new DataTypeTemplatesService(get(xmlDocument).documentElement);
```

Comment:
You should consume the data-types-templates store to retrieve the dataTypeElements directly in this component. You don't need to pass this service from `element-type-container.svelte`

Something like:
```js
const { dataTypeTemplatesSubElements } = dataTypeTemplatesStore

  let columns: Column[] = [
    {
      name: ELEMENT_NAMES.substation,
      visible: true,
      items: dataTypeTemplatesSubElements.substations,
    },
... 
]
```